Content.Shared/Zombies/ZombieComponent.cs

```diff
@@ -92,13 +92,15 @@ public sealed class ZombieComponent : Component
         // Heal on tick
         [DataField("nextTick", customTypeSerializer:typeof(TimeOffsetSerializer))]
         public TimeSpan NextTick;
+        public int SecondsCrit;
```

Comment:
Anything stateful needs to be a datafield or it doesn't work with save and reload properly.